selfdrive/monitoring/dmonitoringd.py

```diff
@@ -31,8 +31,7 @@ def dmonitoringd_thread():
     # Get interaction
     if sm.updated['carState']:
       v_cruise = sm['carState'].cruiseState.speed
```

Comment:
controlsState.vCruise is the superset of PCM and non-PCM cars. This currently doesn't capture Hyundai OP long for example